### PR TITLE
Changed all color of the app

### PR DESCRIPTION
**Pull request by pritam-kr**
- [Feat: Follow and unfollow, latest post of a particular user](https://github.com/pritam-kr/like/pull/7/commits/1c048527c830c2525faa5330065e3f4800b51d5e)
- [Services of follow and unfollow users](https://github.com/pritam-kr/like/pull/7/commits/f3a1176d961c7086a6388cdcf12f56d9278170a0)
- [Customize scroll bar](https://github.com/pritam-kr/like/pull/7/commits/9b1eb21720dbe7b4c78fdcd4956e94df43b6a4dc)
- [Create PostServices.js](https://github.com/pritam-kr/like/pull/7/commits/ad7fda172f792a1779215a73c1182b52c96d2a2e)

### CODE REVIEW

File: src/Components/PostCard/PostCard.jsx [hunk 1]
```diff
@@ -1,31 +1,29 @@
...
   const postDeleteHandler = (_id) => {
     deletedPost(_id);
   };
 
   //Like/Dislike Handler
   const likeDislikeHandler = (_id) => {
-      likePost(_id)
-  }
+    likePost(_id);
+  };
```
Comment (gopalbbharadva) [inline src/Components/PostCard/PostCard.jsx:23]: These both can be write as one liner.

**Review by gopalbbharadva — APPROVED** [review]
Ui looks nice . Good to go.

File: src/Components/PostCard/PostCard.jsx (hunk 1 above)
Comment (pritam-kr) [reply src/Components/PostCard/PostCard.jsx:23]: ok I'll do that

**Review by SukanyaSen530 — APPROVED** [review]
UI is good, liked the color combo 👍🏼

File: src/Components/FollowerModal/FollowingModal.jsx [hunk 2]
```diff
@@ -1,9 +1,14 @@
 import React from "react";
 import * as FaIcons from "react-icons/fa";
+import { useSelector } from "react-redux";
 import {useModalContext} from "../../Context/ModalContext"
 
 const FollowingModal = () => {
 
+  const {user : {users}, auth} = useSelector((state) => state)
+
+  // console.log(auth)
```
Comment (varnita619) [inline src/Components/FollowerModal/FollowingModal.jsx:10]: remove console log

File: src/Components/Index.js [hunk 3]
```diff
@@ -9,3 +9,4 @@ export { FollowerModal } from "./FollowerModal/FollowerModal";
 export {FollowingModal} from "./FollowerModal/FollowingModal"
 export {Loading} from "./Loading/Loading"
 export {Nav} from "./SecondryNav/Nav"
+export {Users} from "./Users/Users"
```
Comment (varnita619) [inline src/Components/Index.js:12]: good use of barrel exports

**Review by varnita619 — APPROVED** [review]
Code is clean and modular
Every possible component is extracted out
Ready to merge

**Pull request merged**